### ISSUE TITLE
Some trouble about structure

### ISSUE THREAD

**Issue by bobo12138** [Adamdad/keras-YOLOv3-mobilenet#11]
Hi,I see your change in the structure of YOLO3 ,perfect! But  I want to  know what need to  modify if I want to change the structure with my mind?

**Comment by Adamdad** [comment]
What would you do on Darknet53

**Comment by bobo12138** [comment]
try some new structure ,  add some conv  lays or  dconv lays

**Comment by Adamdad** [comment]
That's pretty easy. The structure of darknet53 is written in yolo3/model.py. you can do whatever you want on the model.

**Comment by bobo12138** [comment]
here is my e-mail: booookchen@outlook.com   , please

**Comment by bobo12138** [comment]
but  , when I  changed  the  structure  ,  how to  train ?  the weights ???

**Comment by bobo12138** [comment]
I  can't use  the  pre-trained  weights?

**Comment by Adamdad** [comment]
The weight initialization is the key to deep learning. You can initialize the model with different coefficient distribution. To make the model better, pre-train on imagenet is a much powerful way.

**Comment by Adamdad** [comment]
You can not use the existing model. Pre-trained model means trained in advance for a certain structure. You have to train your own if you need

**Comment by bobo12138** [comment]
yes,thankyou. But how to initialize the model? can you tell me the detail? please

**Comment by bobo12138** [comment]
I need just modify the structure  ,and  do not load the pre-trained weights,just  run the train ?

**Comment by Adamdad** [comment]
Here is a good example.
https://stackoverflow.com/questions/48457124/where-is-the-kernel-weight-initialization-in-my-cnn-model
In keras, it helps you to initialize your weight. You can also choose the specific way of initialization for your experiment.

**Comment by bobo12138** [comment]
thank you so much ,you are nice budy!
